src/contentScript.js

```diff
@@ -137,13 +139,13 @@ var __gcrExtAnswers;
     var spanText = document.createElement('text');
     spanText.innerHTML = 'Canned responses ';
 
-    var editButton = createNodeWithClass('button', 'btn-link github-canned-response-edit');
-    editButton.type = 'button';
-    editButton.innerHTML = '(edit or add new)';
-    editButton.addEventListener('click', showEditView);
+    var editLink = createNodeWithClass('a', 'github-canned-response-edit');
+    editLink.href = '#gcr-ext-editor';
+    editLink.setAttribute('rel', 'facebox[.gcrdialog]');
+    editLink.innerHTML = '(edit or add new)';
```

Comment:
Yes I did just change this to button in #12, but this `rel` facebox hook currently doesn't work with buttons, will work 🔜  though.